File: Leetcode/168.-Excel-Sheet-Column-Title/168.py
//chr: int to char
//ord: char to int
def convertToTitle(self, num):
	capitals = [chr(x) for x in range(ord('A'), ord('Z') + 1)];
	result = []
	while n > 0 :
		result.append(capitals[(n - 1) % 26])
		n = (n - 1)// 26
	result.reverse()
	return ''.join(result)

File: Leetcode/168.-Excel-Sheet-Column-Title/168.cpp
string convertToTitle(int n) 
{
	string res;
	while (n > 0)
	{
		char c = ((n - 1) % 26) + 'A';
		res = c + res;
		n = (n - 1) / 26; //Z
	}
    return res;
}
